tests/terminate-scenarios-test.py

```diff
@@ -98,6 +98,13 @@
         errorExit("Failed to relaunch Eos instance")
     Print("nodeos instance relaunched.")
 
+    if maxReversibleBlocks > 0:
+        if not cluster.getNode(0).kill(killSignal):
+            errorExit("Failed to kill Node0")
+        # should immediately exit as max-reversible-blocks set to 2
```

Comment:
You can add a comment saying this is for testing restarting stuck in infinite loop problem.